src/commands/queue.rs

```diff
@@ -141,13 +156,36 @@ fn create_queue_embed<'a>(
     embed.title("Queue");
     embed.field("🔊  Now playing", description, false);
     embed.field("⌛  Up next", build_queue_page(tracks, page), false);
-    embed.footer(|f| {
-        f.text(format!(
-            "Page {} of {} • Requested by {}",
-            page + 1,
-            calculate_num_pages(tracks),
-            author
-        ))
+    embed.footer(|f| f.text(format!("Page {}/{}", page + 1, calculate_num_pages(tracks))));
+
+    embed
+}
+
+fn build_single_nav_btn(label: &str, is_disabled: bool) -> CreateButton {
+    let mut button = CreateButton::default();
+
+    button
+        .custom_id(label.to_string().to_ascii_lowercase())
+        .label(label)
+        .style(ButtonStyle::Primary)
+        .disabled(is_disabled);
+
+    button
```

Comment:
Is this not possible?

```suggestion
    CreateButton::default()
        .custom_id(label.to_string().to_ascii_lowercase())
        .label(label)
        .style(ButtonStyle::Primary)
        .disabled(is_disabled)
```

Reply:
Tried it as well, it returns a `&mut CreateButton`. Could slap it a `.to_owned` but I dunno.

Reply:
I like that